blog/2023-07-13-healthsearch-blogpost/index.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: Discover Healthsearch Unlocking Health with Semantic Search ✨
+slug: healthsearch-demo
+authors: edward
+date: 2023-07-13
+image: ./img/hero.png
+tags: ['concepts','engineering','how-to']
+description: "Learn about our latest open-source demo and how we used Semantic and Generative Search to improve access to health"
+
+---
+
+![Healthsearch Demo](./img/hero.png)
+
+<!-- truncate -->
+
+## 🛠️ Bridging semantic and generative search!
+
+Isn’t it annoying when you just want to find a remedy for your headache and instead find out that you have a horrible disease? With Healthsearch you will find exactly and only that - and no unsolicited health diagnosis.
```

Comment:
> Isn’t it annoying when you just want to find a remedy for your headache and instead find out that you have a horrible disease? With Healthsearch you will find exactly and only that - and no unsolicited health diagnosis.

So what we're saying here is that searching for mostly harmless symptoms related to your health could end to that "damn, after Googling my symptom, I'm pretty sure that I'm going to die in the next 4 days, for sure!"

I would rephrase it a bit to make this clearer, and also exaggerate it a bit more for style and fun.

> Do you know this feeling? You search for something to relieve your headache in the search engine of your choice and end up being sure that you're going to die of a brain tumor within the next few days? The phenomenon is called Dr. Google, but don't be fooled! Just use our Healthsearch demo and find out what users reviewed helped them with whatever it is.